src/FixedDataTableCell.js

```diff
@@ -244,7 +249,15 @@ var FixedDataTableCell = createReactClass({
         <div
           className={cx('fixedDataTableCellLayout/columnResizerContainer')}
           style={columnResizerStyle}
-          onMouseDown={this._onColumnResizerMouseDown}>
+          /**
+           * Stop the event from being propagated when touching the handle.
+           * This prevents the rows from moving around when we resize the headers.
+           */
+          onTouchStart={e => e.stopPropagation()}
+          onTouchEnd={e => e.stopPropagation()}
+          onTouchMove={e => e.stopPropagation()}
```

Comment:
use touchEnabled and only add these listeners if its true.

Reply:
Good call.